validator/src/commands/run/args.rs

```diff
@@ -27,6 +32,28 @@ use {
 const EXCLUDE_KEY: &str = "account-index-exclude-key";
 const INCLUDE_KEY: &str = "account-index-include-key";
 
+#[derive(Debug, PartialEq)]
+pub struct RunArgs {
```

Comment:
Just want to make sure I follow the approach. Is the idea that you're creating this `RunArgs` struct right now, and that you'll gradually add a bunch more fields to the struct ?

Reply:
exactly!